tool/teleport/common/teleport.go

```diff
@@ -391,42 +430,90 @@ func onConfigDump(flags dumpFlags) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
+
+	configPath, err := dumpConfigFile(flags.output, sfc.DebugDumpToYAML(), sampleConfComment)
+	if err != nil {
+		return trace.Wrap(err)
+	}
+
+	if configPath != "" {
```

Comment:
Can returned `configPath` actually be empty if the err is nil?

Reply:
Yes, it is empty when `--output stdout` (which is the default value).